cardano-api/src/Cardano/Api/Orphans.hs

```diff
@@ -278,3 +293,101 @@ instance ToJSON Shelley.RewardType where
 instance ToJSON (SafeHash.SafeHash c a) where
   toJSON = toJSON . SafeHash.extractHash
 
+-----
+
+instance ToJSON Alonzo.ExUnits
+deriving instance FromJSON Alonzo.ExUnits
+
+deriving instance ToJSON Alonzo.Prices
+deriving instance FromJSON Alonzo.Prices
+
+instance ToJSON SBS.ShortByteString where
+  toJSON = Aeson.String . Text.decodeLatin1 . B16.encode . SBS.fromShort
+instance FromJSON SBS.ShortByteString where
+  parseJSON v = case v of
+    Aeson.String b16 ->
+      case B16.decode $ Text.encodeUtf8 b16 of
+        Right decoded -> return $ SBS.toShort decoded
+        Left err -> fail err
+    wrong -> fail $ "Error decoding ShortByteString. \
+                    \Expected a JSON string but got: " <> show wrong
+
+instance FromJSON (MemoBytes (Map Text Integer))
+instance ToJSON (MemoBytes (Map Text Integer))
```

Comment:
How does this instance work?

If we need it, we can and should do a general instance for any `MemoBytes` in terms of its content type by unwrapping it using the appropriate constructor/pattern.

Reply:
`MemoBytes` is defined like this:

```
data MemoBytes t = Memo {memotype :: !t, memobytes :: ShortByteString}
  deriving (NoThunks) via AllowThunksIn '["memobytes"] (MemoBytes t)

deriving instance Generic (MemoBytes t)
```